### PR TITLE
Improve Quality of User Guide

### PR DESCRIPTION
**Pull request by sharpstorm**
## Changes
* Improves standardization of language and formatting across user guide

## Related Issues
* Follows up on #89

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -474,20 +459,19 @@ Examples:
 
 ### Chaining Commands: `&&`
 
-Perform actions on a group of contacts
+Perform multiple actions in a single command.
 
 Format: `COMMAND_A && COMMAND_B`
 
-* Call multiple commands
+* Calls multiple specified scommands
```
Comment (dandaandaaaaaan) [inline docs/UserGuide.md:466]: typo 
```suggestion
* Calls multiple specified commands
```

**Review by dandaandaaaaaan — CHANGES_REQUESTED** [review]
one minor one

File: docs/UserGuide.md (hunk 1 above)
Comment (sharpstorm) [reply docs/UserGuide.md]: Fixed, thank you for the spot.

**Review by Wei-Xinn — APPROVED** [review]
lgtm

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm!

**Pull request merged**